<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1530</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color:rgb(204,229,255)
}

QLineEdit{
background-color:rgb(150,200,255);
border:none
}

QHBoxLayout{
background-color:rgb(204,229,255)
}

QFormLayout{
background-color:rgb(051,153,255);
border:5px
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>70</y>
      <width>271</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="depotwertLabel">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Portfolio Value</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="portValue">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="depotwertLabel_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Asset Base</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="assetBase">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="depotwertLabel_3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>W/L (EUR)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="winLoss">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="depotwertLabel_4">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>W/L (%)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="winLossP">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>100</y>
      <width>391</width>
      <height>251</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>391</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="gLayout"/>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidgetMarket">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>511</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <column>
     <property name="text">
      <string>name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>type</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>price</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>CHG</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>CHG (%)</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tabWidP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>1351</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>120</number>
    </attribute>
    <column>
     <property name="text">
      <string>name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>type</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>numbers</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>buying course</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>asset base</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>current price</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>current asset</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>win/loss (EUR)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>win/loss (%)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>52 Week High</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>52 Week Low</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="depotwertLabel_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>441</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>26</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Stock Market Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="depotwertLabel_6">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Portfolio Details</string>
    </property>
   </widget>
   <widget class="QLabel" name="dateTime">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>10</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>19</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="depotwertLabel_7">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>60</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>Asset Distribution</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>230</y>
      <width>271</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="depotwertLabel_8">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Currencies</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="currencies">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="depotwertLabel_9">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Stocks</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="stocks">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="depotwertLabel_10">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Asia Funds</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="asiafunds">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="depotwertLabel_11">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>World Funds     </string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="worldfunds">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
